catalog-info.yaml

```diff
@@ -128,10 +128,11 @@ spec:
       description: fleet-server environment tests
     spec:
       repository: elastic/fleet-server
-      pipeline_file: .buildkite/pipeline.service-tests.yml
+      pipeline_file: .buildkite/pipeline.service-tests.yaml
       branch_configuration: "main"
       provider_settings:
         build_pull_requests: false
+        trigger_mode: none
       teams:
         cloud-k8s-region:
```

Comment:
I think you want to replace this with your team here.
Otherwise you would not be able to manually invoke and test the pipline afaict